### DISCUSSION TITLE
Unable to get rewards or see my plots in Chia GUI

### DISCUSSION THREAD

**Discussion by Shah-Sahab** [Chia-Network/chia-blockchain#6532] [Support]
Last night created two plots on my local machine. External HDD was full and I was not able to copy my files at the end of the process to an External HDD. I restarted Chia GUI in the hopes of rewriting these plots on another Harddrive but Inow Chia GUI does not load my Plots, even though I can see them saved in my local folder. How can I load these plots to Chia/Chia GUI and earn the rewards for them?

**Comment by loppefaaret** [comment]
You would have to manually rename the files in the temp folder from .plot.2.tmp to .plot, and then have them in a folder that is added to the plots folders

**Comment by BrandtH22** [comment]
@Shah-Sahab , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
